src/test/java/uk/sky/cqlmigrate/LockVerificationTest.java

```diff
@@ -36,13 +47,22 @@
     private static final String KEYSPACE = "locker";
     private static String TABLE_NAME = KEYSPACE + ".lock_testing";
 
-    private Cluster cluster;
-    private Session session;
+    private static Server server = Server.builder().build();
+    private ClusterSpec cluster = ClusterSpec.builder().build();
+    private BoundCluster bCluster;
```

Comment:
Now that this test uses simulacron rather than real cassandra, could the @ignore be removed? Can it run in pipelines now?

Reply:
This test doesn't pass on master with simulacron even on a local run for me. So I think this needs to be looked at as a separate issue.

Reply:
this class was already commented out and might need lots of changes. should we better raise another issue for it?
if it was a migration to simulacron change, it should had been in the scope of that card anyway
how do you feel about it?

Reply:
If it's not passing and already ignored, we can push this requirement out of this PR, you're right.